src/Microsoft.AspNetCore.ResponseCaching/ResponseCacheContext.cs

```diff
@@ -32,21 +32,21 @@ public class ResponseCacheContext
 
         public HttpContext HttpContext { get; }
 
-        public bool ShouldCacheResponse { get; internal set; }
+        public DateTimeOffset ResponseTime { get; internal set; }
```

Comment:
Should we make this value type nullable? Otherwise the default value might be confusing. Same with CachedEntryAge